### PR TITLE
feat(demo): releases and quick organization

### PR DESCRIPTION
**Pull request by scefali**
This PR makes threee changes for demo mode:
1. Adds a series of releases corresponding to the timing of generated errrors. It is roughly based on `load-mock` but with some improvements to make things more "real". Firstly, I have configured it so we can get a suspect commit for each of the issues we generate. Secondly, I have updated commit messages and randomly generated fil paths to seem a bit more realistic.
Screen Shot 2021-03-22 at 8 43 28 AM
Screen Shot 2021-03-22 at 8 43 41 AM

2. I have changed how we handle the empty organization buffer case (no orgs with populated data ready). Rather than generating an error or putting the user on some waiting page, instead, we will quickly spin up an organization in a synchronous manner with far fewer events. This is a sub-optimal experience, but it's less bad than making the user minutes for a "full" organization to be spun up. It's unclear how often this might happen (or even if it might happen), but it's a decent way for us to automatically scale up for increased localized demand.

3. Small change to not set the `accepted_tracking` cookie except if the value passed in is a `1` or `0`.

### CODE REVIEW

File: src/sentry/demo/data_population.py [hunk 1]
```diff
@@ -99,22 +127,143 @@ def get_user_by_id(id_0_offset):
...
     id_0_offset = random.randrange(0, len(name_list), NAME_STEP_SIZE)
     return get_user_by_id(id_0_offset)
 
 
-def safe_send_event(data):
+def gen_random_author():
+    return (
+        "{} {}".format(random.choice(loremipsum.words), random.choice(loremipsum.words)),
+        "{}@example.com".format(random.choice(loremipsum.words)),
+    )
```
Comment (wmak) [inline src/sentry/demo/data_population.py:148]:
```suggestion
    author = "{} {}".format(random.choice(loremipsum.words), random.choice(loremipsum.words)),
    return (
        author
        "{}@example.com".format(author.replace(" ", ".")),
    )
```

File: src/sentry/demo/data_population.py [hunk 2]
```diff
@@ -106,22 +134,143 @@ def get_user_by_id(id_0_offset):
...
+    config = get_config(quick)
+    NUM_RELEASES = config["NUM_RELEASES"]
+    MAX_DAYS = config["MAX_DAYS"]
+    release_time = timezone.now() - timedelta(days=MAX_DAYS)
+    hourly_release_cadence = MAX_DAYS * 24.0 / NUM_RELEASES
+    org = projects[0].organization
+    org_id = org.id
+    for i in range(NUM_RELEASES):
+        release = Release.objects.create(
+            # version=sha1(uuid4().bytes).hexdigest(),
```
Comment (wmak) [inline src/sentry/demo/data_population.py:203]: Was this intentionally left here?

**Review by wmak — APPROVED** [review]
LGTM, one little touchup around the author generation and a question about a comment that might be leftover code

File: src/sentry/demo/data_population.py (hunk 2 above)
Comment (scefali) [reply src/sentry/demo/data_population.py:204]: @wmak nope, good catch!

**Pull request merged**